src/components/ValidatedInput/ValidSelect.tsx

```diff
@@ -57,13 +59,15 @@ export const ValidSelect = ({
   className,
   dataQa,
   helperText,
+                              noOptionsMessage = () => translation("common.noOptions"),
```

Comment:
how about doing it in `Select`? so that other components that use `Select` will be affected